larq_zoo/training/multi_stage_experiments.py

```diff
@@ -0,0 +1,257 @@
+import tensorflow as tf
+from zookeeper import ComponentField, Field, cli, task
+
+from larq_zoo.literature.real_to_bin_nets import (
+    RealToBinNetBANFactory,
+    RealToBinNetBNNFactory,
+    RealToBinNetFPFactory,
+    ResNet18FPFactory,
+    StrongBaselineNetBANFactory,
+    StrongBaselineNetBNNFactory,
+)
+from larq_zoo.training.datasets import ImageNet
+from larq_zoo.training.knowledge_distillation.multi_stage_training import (
+    LarqZooModelTrainingPhase,
+    MultiStageExperiment,
+)
+
+# --------- Learning rate schedules -------------
```

Comment:
It might be nice to split these (or at least the `CosineDecayWithWarmup`) out into a separate `learning_rates.py` or `utils.py` file under `larq_zoo.training`, since they're useful beyond just R2B.

Reply:
Done in be71e12